curriculum/js1/3.js

```diff
@@ -5,8 +5,13 @@
  * @returns {string}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (x) => {
+   if (x < 0 || x === 0){
+     return "";
+   }
+   if (x > 0){
```

Comment:
no need for if case, because if (x<0 || x ===0), the only possible case is x>0, so you don't need an if